keyform/views.py

```diff
@@ -7,22 +7,29 @@
 from keyform.models import Request
 from django.urls import reverse_lazy
 from django.http import HttpResponseRedirect
+from django.contrib.auth.mixins import LoginRequiredMixin
 
-class HomeView(ListView):
+class HomeView(LoginRequiredMixin, ListView):
     model = Request
     template_name = "keyform/home.html"
+    login_url = "login"
+    redirect_field_name = "redirect_to"
```

Comment:
`login_url` and `redirect_field_name` aren't necessary here (or in any of the following view classes)